db/migrations/02_create_test_database.sql

```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE makersbnb_test ;
```

Comment:
Do not need this file. can just be a note in the readme

Reply:
TY done